docs/source/index.rst

```diff
@@ -5,40 +5,40 @@ StreamAlert
   :align: center
   :alt: StreamAlert
 
-StreamAlert is a serverless, realtime data analysis framework which empowers you to ingest, analyze, and alert on data from any environment, using datasources and alerting logic you define.
+StreamAlert is a serverless, real-time data analysis framework which empowers you to ingest, analyze, and alert on data from any environment, using data sources and alerting logic you define.
+Security teams use StreamAlert to scan terabytes of log data every day for proactive threat detection and prevention.
 
-For more details, see our `announcement post <https://medium.com/@airbnbeng/e8619e3e5043>`_.
-
-At a high-level
----------------
-
-* Deployment is automated: simple, safe and repeatable for any AWS account
-* Easily scalable from megabytes to terabytes per day
-* Infrastructure maintenance is minimal, no devops expertise required
-* Infrastructure security is a default, no security expertise required
-* Supports data from different environments (ex: IT, PCI, Engineering)
-* Supports data from different environment types (ex: Cloud, Datacenter, Office)
-* Supports different types of data (Ex: JSON, CSV, Key-Value, and Syslog)
-* Supports different use-cases like security, infrastructure, compliance and more
-
-Components
-----------
+Overview
+--------
+Incoming log data is classified and scanned by the StreamAlert rules engine running in your AWS account.
+Rule matches are reported to one or more alert outputs:
 
 .. figure:: ../images/sa-high-level-arch.png
```

Comment:
Can not add comment to the binary file, so I add it here.
* I would prefer simpler diagram, replacing `Alert Delivery` with `Output` and only includes PagerDuty, Slack, Amazon S3, AWS SNS.
* It is high-level architecture, including CloudWatch logs, kinesis firehose SNS, SQS is too detailed.

Reply:
I like listing at least all of the AWS alert targets because it demonstrates just how powerful the alerting mechanism can be. Someone looking at StreamAlert for the first time can immediately see that they could use alerts to trigger their own Lambda functions, feed their own analysis SQS queues or Firehoses, etc.

I like `Alert Delivery` instead of `Output` because the connotation of an *alert* is something anomalous or noteworthy, as opposed to an "output", which is just the result of an execution. Most StreamAlert invocations will not generate any alerts, so calling it an "output" feels like a misnomer to me.

But I'm happy to discuss further. @ryandeivert what are your thoughts on the diagram?

Reply:
![sa-high-level-arch](https://user-images.githubusercontent.com/3608925/43489341-937925dc-94d1-11e8-9f41-b4ecf97a237d.png)

Reply:
I'm fairly indifferent on this, but do see how showing the AWS ones is helpful, and slightly prefer alert delivery to output. that said, the amount of alert output services is going to continue grow, so trying to include all is going to be hard over time. ideally, we'd highlight 4 that serve different purposes:

1) Slack - chatops style message
2) PD - case/incident management
3) S3 - historical retention
4) SQS - downstream tooling

just my 2 cents